web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -334,16 +335,16 @@ was not inplace, since it is possible to go out of memory because of this.
 
 #### Return the calling object (`self`) also when using `inplace=True`?
 
-The downsides of keeping the `inplace=True` option for certain methods, are that the return type of those methods will
-now depend on the value of `inplace`, and that method chaining will no longer work.
-
-One way around this is to have the method return the original object that was operated on inplace when `inplace=True`.
+One of the downsides of the `inplace=True` option is that the return type of those methods
+depends on the value of `inplace`, and that method chaining does not work.
+Those downsides are still relevant for the cases where we keep `inplace=True`. Therefore,
+it is also an option to have the method return the original object that was operated
+on inplace when `inplace=True`.
 
 Advantages:
 
 - It enables to use inplace operations in a method chain
 - It simplifies type annotations
-- It enables to change the default for `inplace` to True under Copy-on-Write
```

Comment:
I removed this advantage, because I am not sure this is actually true? (I don't recall our reasoning exactly)

Theoretically we _can_ change the default of course, because it doesn't violate CoW since it returns an indentical object and not a new object. But I am not sure we would actually consider doing that? It would be strange to change to not return a new object for just a few methods?

Reply:
+1 on the removal here.